lms/resources/lti_launch.py

```diff
@@ -101,7 +102,13 @@ def canvas_sections_supported(self):
             # Canvas course sections feature was released.
             return False
 
-        return self._request.find_service(name="ai_getter").canvas_sections_supported()
+        application_instance_service = self._request.find_service(
+            name="application_instance"
+        )
+        try:
+            return bool(application_instance_service.get().developer_key)
```

Comment:
```suggestion
        try:
            return bool(self._request.find_service(name="application_instance").get().developer_key)
```

Not sure if this is better or worse? Depends on how `black` handles it?